src/combat.ts

```diff
@@ -53,6 +55,67 @@ export class Macro extends LibramMacro {
     return new Macro().tryHaveSkill(skillOrName);
   }
 
+  tryCopier(itemOrSkill: Item | Skill, monster: Monster = $monster`none`): Macro {
+    switch (itemOrSkill) {
+      case $item`Spooky Putty sheet`:
+        return this.externalIf(
+          get("spookyPuttyCopiesMade") < 5 && !have($item`Spooky Putty monster`),
```

Comment:
This should also check to make sure `spookyPuttyCopiesMade + _raindohCopiesMade < 6`, shouldn't it? And same for below with black box.